osu.Game.Rulesets.Catch/Objects/Drawables/CaughtObject.cs

```diff
@@ -19,7 +17,7 @@ namespace osu.Game.Rulesets.Catch.Objects.Drawables
     [Cached(typeof(IHasCatchObjectState))]
     public abstract partial class CaughtObject : SkinnableDrawable, IHasCatchObjectState
     {
-        public PalpableCatchHitObject HitObject { get; private set; }
+        public PalpableCatchHitObject HitObject { get; private set; } = null!;
```

Comment:
This is a pooled drawable and HitObject is non-null while active (after obtained from a pool).
(I forgot this file in the first commit)